README.md

```diff
@@ -75,27 +46,22 @@ source .venv/bin/activate
 uv pip install notte-sdk
 ```
 
-...and spin up your crazy cool and dead simple agent on our remote machine:
+And spin up your crazy cool and dead simple agent on our remote machine:
 
 ```python
 import os
 from notte_sdk import NotteClient
 
-notte = NotteClient(api_key=os.getenv("NOTTE_API_KEY"))
+client = NotteClient(api_key=os.getenv("NOTTE_API_KEY"))
```

Comment:
Could you keep `notte` instead of `client` it's used consistently everywhere else (docs, readmes, tests, etc.)